### PR TITLE
Fixes #28314 - fixes autocomplete_search

### PR DESCRIPTION
**Pull request by ezr-ondrej**
~~Would be perfect to have theforeman/foreman#7189 first, but this works without it.~~
Edit - it's a hard dependency now.
- [x] merge theforeman/foreman#7189

### CODE REVIEW

File: app/views/job_invocations/_tab_hosts.html.erb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <% if job_invocation.resolved? %>
-  <%= form_tag job_invocation_path(job_invocation), :method => "get",:id =>"search-form" do %>
+  <%= form_with url: job_invocation_path(job_invocation), method: "get", id: "search-form" do |f| %>
     <div class="row">
       <div class="title_filter col-md-6">
         <div class="input-group">
-            <%= auto_complete_search(:search, params[:search].try(:squeeze, " "), :placeholder => _("Filter") + ' ...', :path => hosts_path) %>
+            <%= autocomplete_f(f, :search, value: params[:search].try(:squeeze, " "), placeholder: _("Filter") + ' ...', path: hosts_path, error: []) %>
```
Comment (ezr-ondrej) [inline app/views/job_invocations/_tab_hosts.html.erb]: The `error: []` is a hack to get over what we miss in core. 
```suggestion
            <%= autocomplete_f(f, :search, value: params[:search].try(:squeeze, " "), placeholder: _("Filter") + ' ...', path: hosts_path) %>
```

**Comment by adamruzicka** [comment]
This doesn't look right
image

**Comment by ares** [comment]
Do we still need this? What's the status?

**Comment by ezr-ondrej** [comment]
We do, the core PR is still not in though.

**Comment by adamruzicka** [comment]
The core PR is in now

**Comment by adamruzicka** [comment]
Thank you @ezr-ondrej !

**Pull request merged**